stylesheet.css

```diff
@@ -1,146 +1,155 @@
-body{
-    background-color: #161316;
-}
-
-.mainBannerContainer{
-    position: auto;
-    display: block;
-    width: 1300px;
-    height: 780px;
-    margin-top: 20px;
-    margin-left:auto;
-    margin-right:auto;
-    margin-bottom: 0;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
This is a nice reset. It lets us style our padding and margin without any defaults.